arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "version": "2017-06-01",
-    "title": "StorSimpleSeries8000ManagementClient"
+    "title": "StorSimple8000SeriesManagementClient"
```

Comment:
Yes, this is intentional change. At all other places we have used the Keyword StorSimple8000Series, and hence wanted to be consistent with it here too.